torch/onnx/symbolic_opset9.py

```diff
@@ -1594,3 +1594,15 @@ def log2(g, self):
 
 def prim_shape(g, self):
     return g.op('Shape', self)
+
+
+@parse_args('v', 'i', 'v', 'v')
+def gather(g, self, dim, index, sparse_grad=False):
+    # NOTE: Update this workaround if ONNX has native Gather support.
```

Comment:
Maybe rephrase to say that current onnx::Gather does not directly support torch.Gather.